### ISSUE TITLE
The same number of AB and GD chains

### ISSUE THREAD

**Issue by MaryGoAround** [ncborcherding/scRepertoire#160]
Hi

When combining contig list from different samples, I noticed I have the same number of AB and GD chains 

```
> combined <- combineTCR(contig_list, 
+                        samples = c("sample1", "sample2"), 
+                        ID = c("p", "p"),cells ="T-AB")

> dim(combined$sample1_p)
[1] 2848   14
> dim(combined$sample2_p)
[1] 3224   14b

> combined <- combineTCR(contig_list, 
+                        samples = c("sample1", "sample2"), 
+                        ID = c("p", "p"),cells ="T-GD")

> dim(combined$sample1_p)
[1] 2848   14
> dim(combined$sample2_p)
[1] 3224   14b

```
I tried built in data and got the same number

```
> combined <- combineTCR(contig_list, 
                         +                        samples = c("PY", "PY", "PX", "PX", "PZ","PZ"), 
                         +                        ID = c("P", "T", "P", "T", "P", "T"), cells ="T-GD")
> dim(combined$PY_P)
[1] 3531   14

> combined <- combineTCR(contig_list, 
                         +                        samples = c("PY", "PY", "PX", "PX", "PZ","PZ"), 
                         +                        ID = c("P", "T", "P", "T", "P", "T"), cells ="T-AB")
> dim(combined$PY_P)
[1] 3531   14
>

> quantContig(combined, cloneCall="gene+nt", chain = "TRA")
```

![Rplot13](https://user-images.githubusercontent.com/60115705/167431643-a88c90af-8904-4238-b13b-99bb75985ffe.png)

I am not able to get the point here, how I can get the GD chain?

Thanks for any help

**Comment by ncborcherding** [comment]
Hey MaryGoRound, 

Thanks for reaching out.

I would expect the same number of resulting from "T-AB" and "T-GD" as the ```combineTCR()``` function is barcode-based. This is to say it will make the same data frame for each sample, but only look at the specific chains (alpha/beta vs gamma/delta) - the other chains will be listed as NAs. 

tl;dr: No, the same number is not anything wrong. 

Nick
